hnn_core/cell.py

```diff
@@ -325,10 +325,12 @@ def _create_sections(self, p_secs, topology):
             self.sections[sec_name] = sec
 
             h.pt3dclear(sec=sec)
+            h.pt3dconst(0, sec=sec)  # be explicit, see documentation
             for pt in p_secs[sec_name]['sec_pts']:
                 h.pt3dadd(pt[0] + dx,
                           pt[1] + dy,
                           pt[2] + dz, 1, sec=sec)
+            # with pt3dconst==0, these will alter the 3d points defined above!
```

Comment:
So I suppose this essentially functions to define the direction in which the section points?

Reply:
That _must_ be it, yes, though the documentation manages to be pretty unclear...